src/tinypeel/tinypeel.py

```diff
@@ -238,6 +238,36 @@ def get_probability_options():
     return parse_dictionary
 
 
+def get_input_options():
+    parse_dictionary = dict()
+    parse_dictionary["bfile"] = lambda parser: parser.add_argument('-plink_file', default=None, required=False, type=str,
+                                                                   nargs="*",
+                                                                   help='A file in plink (binary) format. Only stable on Linux).')
+    parse_dictionary["genotypes"] = lambda parser: parser.add_argument('-geno_file', default=None, required=False,
+                                                                       type=str, nargs="*",
+                                                                       help='A file in AlphaGenes format.')
+    parse_dictionary["reference"] = lambda parser: parser.add_argument('-reference', default=None, required=False,
+                                                                       type=str, nargs="*",
+                                                                       help='A haplotype reference panel in AlphaGenes format.')
+    parse_dictionary["seqfile"] = lambda parser: parser.add_argument('-seq_file', default=None, required=False, type=str,
+                                                                     nargs="*", help='A sequence data file.')
+    parse_dictionary["pedigree"] = lambda parser: parser.add_argument('-ped_file', default=None, required=False,
+                                                                      type=str, nargs="*",
+                                                                      help='A pedigree file in AlphaGenes format.')
+    parse_dictionary["phasefile"] = lambda parser: parser.add_argument('-phasefile', default=None, required=False,
```

Comment:
@AprilYUZhang I think we have missed this in our meeting - in line with the https://github.com/AlphaGenes/AlphaPeel/issues/89 we should call this `-hap_file`

Reply:
Ok, I will add it.